### ISSUE TITLE
Documentation for how to run the generator on our own infrastructure.

### ISSUE THREAD

**Issue by alanhoyle** [MaayanLab/biojupies#67]
One of the researchers here asked about your BioJupies service and was wondering if it would be possible to run the notebook generator software supported by our own local services and cloud infrastructure, rather than uploading to the http://biojupies.cloud/ site.  

I found the maayanlab/notebook-generator-server-dev and maayanlab/biojupies-dev docker images at DockerHub, and found what seems to be the local Dockerfile in the server subdirectory, but I wasn't able to find documentation about how to set them up to run locally and/or use our own GCP infrastructure.  

We currently have our own local installations of JupyterHub and run some other Docker-based local services, and have familiarity with using GCP for compute and object storage. 

Is it simply a matter of doing something like this? 

```
docker run -e APPLICATION_DEFAULT_CREDENTIALS -e GOOGLE_APPLICATION_CREDENTIALS  maayanlab/notebook-generator-server-dev
```

And exposing some ports to the container?  If so, what ports need to be exposed?

**Comment by EngineerReversed** [comment]
Yes, I would like to know the same as well. We have our own infrastructure and we want to generate notebooks by using our own data and services. Is it possible to do so?
